<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(69, 45, 91, 255), stop:0.719212 rgba(35, 40, 84, 255));
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame_ave_temp">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,30%);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ave_temp_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
font: &quot;Helvetica&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>
&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p/&gt;&lt;p&gt;Average &lt;br/&gt;Temperature&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="ave_temp_text">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>12</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font: 24pt bold;
</string>
     </property>
     <property name="plainText">
      <string>22.33
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>50</width>
       <height>4</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: orange;
height: 8pt</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="ave_temp_unit_text">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>12</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font: 12pt bold;
</string>
     </property>
     <property name="plainText">
      <string>°C</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_ave_temp_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,30%);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="high_temp_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
font: &quot;Helvetica&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Highest &lt;br/&gt;Temperature&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="high_temp_text">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>12</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font: 24pt bold;
</string>
     </property>
     <property name="plainText">
      <string>22.33
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>72</width>
       <height>4</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: lightblue;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="high_temp_unit_text">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>12</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font: 12pt bold;
</string>
     </property>
     <property name="plainText">
      <string>°C</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_ave_temp_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,30%);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="low_temp_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
font: &quot;Helvetica&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Lowest &lt;br/&gt;Temperature&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="low_temp_text">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>12</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font: 24pt bold;
</string>
     </property>
     <property name="plainText">
      <string>22.33
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>20</width>
       <height>4</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: lightgrey;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="low_temp_unit_text">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>12</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font: 12pt bold;
</string>
     </property>
     <property name="plainText">
      <string>°C</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_ave_temp_6">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,30%);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="low_hum_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
font: &quot;Helvetica&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Lowest &lt;br/&gt;Humidity&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="low_hum_text">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>12</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font: 24pt bold;
</string>
     </property>
     <property name="plainText">
      <string>10.1
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>20</width>
       <height>4</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: pink;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="low_hum_unit_text">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>12</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font: 12pt bold;
</string>
     </property>
     <property name="plainText">
      <string>%</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_ave_temp_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,30%);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="high_hum_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
font: &quot;Helvetica&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Highest &lt;br/&gt;Humidity&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="high_hum_text">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>12</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font: 24pt bold;
</string>
     </property>
     <property name="plainText">
      <string>70.4
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>72</width>
       <height>4</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: green;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="high_hum_uni_text">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>12</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font: 12pt bold;
</string>
     </property>
     <property name="plainText">
      <string>%</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_ave_temp_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,30%);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ave_hum_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
font: &quot;Helvetica&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Average &lt;br/&gt;Humidity&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="ave_hum_text">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>12</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font: 24pt bold;
</string>
     </property>
     <property name="plainText">
      <string>35.3
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>50</width>
       <height>4</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: darkblue;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="ave_hum_unit_text">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>12</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font: 12pt bold;
</string>
     </property>
     <property name="plainText">
      <string>%</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,30%);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="current_temperature_text">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 60pt;
color: white;
background: transparent</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="plainText">
      <string>22.44</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="current_humidity_text">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 60pt;
color: white;
background: transparent</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="plainText">
      <string>34.5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_unit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>19</red>
           <green>140</green>
           <blue>252</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>70</red>
           <green>25</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>70</red>
           <green>25</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>19</red>
           <green>140</green>
           <blue>252</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>70</red>
           <green>25</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>70</red>
           <green>25</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>19</red>
           <green>140</green>
           <blue>252</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>70</red>
           <green>25</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>70</red>
           <green>25</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(70,25,255);
</string>
     </property>
     <property name="text">
      <string>°C &lt;-&gt; °F</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="current_temp_unit_text">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>12</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font: 24pt bold;
</string>
     </property>
     <property name="plainText">
      <string>°C</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="current_hum_unit_text">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>12</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font: 24pt bold;
</string>
     </property>
     <property name="plainText">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLabel" name="temperature_icon">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>60</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../python_program_server/ui_img/thermometer.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="humidity_icon">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>150</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../python_program_server/ui_img/drop.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_location">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>291</width>
      <height>79</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 50pt;
color: white;
background: transparent</string>
    </property>
    <property name="plainText">
     <string>Boulder, CO</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;
color: white;
background: transparent</string>
    </property>
    <property name="plainText">
     <string>Database Connection:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_Database">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;
color: green;
background: transparent</string>
    </property>
    <property name="plainText">
     <string>CONNECTED</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="request_stamp_text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: 12pt;
color: lightgrey;</string>
    </property>
    <property name="plainText">
     <string>last request:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;
color: white;
background: transparent</string>
    </property>
    <property name="plainText">
     <string>Server:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_Server">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;
color: green;
background: transparent</string>
    </property>
    <property name="plainText">
     <string>RUNNING</string>
    </property>
   </widget>
   <zorder>frame_ave_temp</zorder>
   <zorder>frame_ave_temp_2</zorder>
   <zorder>frame_ave_temp_3</zorder>
   <zorder>frame_ave_temp_6</zorder>
   <zorder>frame_ave_temp_5</zorder>
   <zorder>frame_ave_temp_4</zorder>
   <zorder>frame</zorder>
   <zorder>plainTextEdit_location</zorder>
   <zorder>plainTextEdit_2</zorder>
   <zorder>plainTextEdit_Database</zorder>
   <zorder>request_stamp_text</zorder>
   <zorder>plainTextEdit_5</zorder>
   <zorder>plainTextEdit_Server</zorder>
   <zorder>current_temperature_text</zorder>
  </widget>
  <action name="actionTemp">
   <property name="text">
    <string>Temp</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
